support-frontend/assets/components/thankYou/thankYouModuleData.tsx

```diff
@@ -111,8 +112,9 @@ const defaultSupportReminder = {
 const defaultFeedbackSurveyHasBeenCompleted = false;
 
 export const getThankYouModuleData = (
-	countryId: IsoCountry,
+	productKey: ActiveProductKey,
```

Comment:
Might be good to refactor a number of these props so they use this `productKey` (and `ratePlan`/`countryGroupId`) in future. Think outside scope of this PR but for a future cleanup PR would say.